docs/curation-review-tut.md

```diff
@@ -0,0 +1,570 @@
+
+---
+
+## Schema definition
+
+<br/>
+
+The purpose of the cellxgene schema is to support the construction of a data corpus that facilitates data integration across multiple tissues and experiments. This goal requires that we collect a standardized set of metadata about single cell datasets that are uploaded to the cellxgene data portal. To make this process easy to adhere to, we only require a few fields (detailed below) to support easy search and integration across datasets. These metadata are stored within the `AnnData` object (in `adata.uns` and `adata.obs`). To access a more comprehensive decription about our schema requirements, you can refer to the [official schema definition](corpora_schema.md).
+
+In this section, we are covering 1) what metadata are required to adhere to the cellxgene schema and 2) the locations of these metadata in the `AnnData` object. While it is possible to build an object that is acceptable by the cellxgene schema manually, in the [next section](#cellxgene-curation-tools), we will show you how to perform this curation with tools provided by the CZI curation team.
+
+#### Basic requirements ([expanded version](corpora_schema.md#basic-requirements))
+- **Unique observation identifiers:** Each observation (usually a cell) must have an id that is unique within the dataset.
+- **Unique feature identifiers:** Every feature (usually a gene or transcript) also needs a unique identifier.
+- **No PII:** No metadata can be personally identifiable: no names, dates of birth, specific locations, etc. There's a [list](https://docs.google.com/document/d/1nlUuRiQ8Awo_QsiVFi8OWdhGONjeaXE32z5GblVCfcI/edit?usp=sharing).
+
+#### `uns`
+
+`adata.uns` is a dictionary of key-value pairs that we use to store dataset level metadata. This is also where you store information on the different data representations that you are sharing (i.e you are sharing a raw counts matrix, normalized expression matrix, scaled and centered expression matrix)
+
+<br/>
+
+<div align="center">
+
+| Key                                      | Value      | Description                                                                  | Example                 |
+| ---------------------------------------- |:----------:| :----------------------------------------------------------------------------| ------------------------|
+| `adata.uns['version']`                   | string     | schema version (current version is `1.1.0`)                               | `1.1.0`           |
+| `adata.uns['title']`                     | string     | title of publication (and title of dataset if more than one is being submitted) | `An Atlas of Gene Regulatory Elements in Adult Mouse Cerebrum: GABAergic neurons`|
+| `adata.uns['publication_doi']`           | string     | DOI of preprint or official publication             | `https://doi.org/10.1101/2020.05.10.087585` |
+| `adata.uns['organism']`                  | string     | name of organism (first letter capitalized) | `Mouse` |
```

Comment:
The label associated with NCBI Taxon 10090 is Mus musculus, not Mouse. If you're not requesting the species name, I would suggest using an enum.

Reply:
@pablo-gar can you spot check me here? after the schema is applied, `adata.uns` is populated with organism name rather than species name, right?

Reply:
trouble area, may result in inconsistent submissions 

put a note - 'for organisms, options....{human, mouse}'